Applications/Classification/jni/main_train.cpp

```diff
@@ -0,0 +1,395 @@
+/**
+ * Copyright (C) 2019 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ * @file	main_train.cpp
+ * @date	21 April 2020
+ * @see		https://github.com/nnstreamer/nntrainer
+ * @author	Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug		No known bugs except for NYI items
+ * @brief	This is Classification Example with one FC Layer
+ *              The base model for feature extractor is mobilenet v2 with
+ * 1280*7*7 feature size. It read the Classification.ini in res directory and
+ * run according to the configureation.
+ *
+ */
+
+#include "bitmap_helpers.h"
+#include "tensorflow/contrib/lite/interpreter.h"
+#include "tensorflow/contrib/lite/kernels/register.h"
+#include "tensorflow/contrib/lite/model.h"
+#include "tensorflow/contrib/lite/string_util.h"
+#include "tensorflow/contrib/lite/tools/gen_op_registration.h"
+#include <cmath>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <queue>
+#include <sstream>
+#include <stdlib.h>
+#include <time.h>
+
+#include "databuffer.h"
+#include "layers.h"
+#include "neuralnet.h"
+#include "tensor.h"
+
+/**
+ * @brief     Data size for each category
+ */
+#define TOTAL_TRAIN_DATA_SIZE 100
+
+#define TOTAL_VAL_DATA_SIZE 10
+
+#define TOTAL_TEST_DATA_SIZE 100
+
+#define BUFFER_SIZE 100
+
+/**
+ * @brief     Number of category : Three
+ */
+#define TOTAL_LABEL_SIZE 10
+
+/**
+ * @brief     Number of Test Set
+ */
+#define TOTAL_TEST_SIZE 8
+
+/**
+ * @brief     Max Epoch
+ */
+#define ITERATION 3000
+
+#define MINI_BATCH 32
+
+#define training true
```

Comment:
Q. Is there any reason this is in lower case?

Reply:
I will fix it in next PR. Thanks.

Reply:
This going to be updated incoming PR. Thanks.